forch/forch_metrics.py

```diff
@@ -110,6 +110,10 @@ def _add_vars(self):
             'static_mac_vlan', 'MAC addresses with statically assigned vlan', Gauge,
             labels=['mac'])
 
+        self._add_var(
+            'system_initialization', 'Initialization state of the system', Gauge,
+            labels=['authentication'])
```

Comment:
Instead of "authentication" I'd name/label this "auth_mode"

Reply:
Update. PTAL..